examples/passing/DctorName.purs

```diff
@@ -0,0 +1,24 @@
+module Main where
+
+import Prelude
+import Control.Monad.Eff.Console (log)
+
+newtype Bar' = Bar' Int
+
+data Foo' = Foo' Bar'
+
+data Baz'' = Baz'' | Baz'
+
+f :: Foo' -> Boolean
+f a = case a of Foo' b -> true
+
+g :: Baz'' -> Int
+g Baz'' = 0
+g Baz' = 1
+ 
+h :: Bar' -> Int
```

Comment:
Could you please make sure these cases actually get run? You could do this by adding additional top-level declarations of type `Int` for example, where you apply each function.